pages/components/card/card.html

```diff
@@ -0,0 +1,520 @@
+              src="https://carbon.now.sh/embed?bg=rgba%28195%2C216%2C236%2C1%29&t=panda-syntax&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522vertical-card-wrapper%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522vertical-card-image-wrapper%2522%253E%250A%2520%2520%2520%2520%253Cimg%250A%2520%2520%2520%2520%2520%2520class%253D%2522responsive-image%2522%250A%2520%2520%2520%2520%2520%2520src%253D%2522..%252F..%252F..%252Fassets%252Fkeyboard.jpg%2522%250A%2520%2520%2520%2520%2520%2520alt%253D%2522Keyboard%2520image%2522%250A%2520%2520%2520%2520%252F%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522vertical-card-header%2522%253E%250A%2520%2520%2520%2520%253Ch2%2520class%253D%2522m-2%2522%253EKeychron%2520K6%253C%252Fh2%253E%250A%2520%2520%2520%2520%253Csmall%2520class%253D%2522mx-2%2522%253EBlue%2520Switches%253C%252Fsmall%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522vertical-card-body%2520my-4%2520mx-2%2522%253E%250A%2520%2520%2520%2520%253Cp%253E%250A%2520%2520%2520%2520%2520%2520The%2520K6%2520has%2520included%2520keycaps%2520for%2520both%2520Windows%2520and%2520Mac%2520operating%250A%2520%2520%2520%2520%2520%2520systems.%2520Hot-swappable%2520version%2520comes%2520with%2520Gateron%2520switch%250A%2520%2520%2520%2520%2520%2520pre-installed.%250A%2520%2520%2520%2520%253C%252Fp%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522vertical-card-price-wrapper%2520mt-2%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522mb-4%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522product-discount-price%2522%253ERs.%25207%252C999.00%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522product-original-price%2520ml-5%2522%253ERs.%25209%252C438.82%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522product-discount%2522%253ESave%252018%2525%253C%252Fspan%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522vertical-card-footer%2522%253E%250A%2520%2520%2520%2520%253Cbutton%2520type%253D%2522button%2522%2520class%253D%2522btn%2520btn-primary%2520add-btn%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-shopping-cart%2520mr-2%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520Add%2520to%2520Cart%250A%2520%2520%2520%2520%253C%252Fbutton%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+              style="
+                width: 620px;
+                height: 688px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
```

Comment:
viresh suggested me to make a class and avoid inline CSS even for iframes. But if you are facing issues then let it be, carbon troubled me so i switched.

Reply:
Initially, I was also using the same approach but it was breaking in certain use-cases so need to switch back to inline CSS for iFrames.